### ISSUE TITLE
Use different non-specific assay for schema example

### ISSUE THREAD

**Issue by jahilton** [chanzuckerberg/single-cell-curation#190]
The [obs.assay_ontology_term_id schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/schema.md#assay_ontology_term_id) , notes that text can be appended to EFO terms for added specificity, and gives the example of sci-plex as an assay without a specific EFO term.
However, sci-Plex now has an EFO term ([EFO:0030026](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0030026)). So a new example is needed.

**Comment by brianraymor** [comment]
When we update the pinned EFO ontology ...

**Comment by jahilton** [comment]
Actually, going to change this and suggest that we remove this option to add the specification, thus removing the need for an example.
This is inconsistent with how we manage the other ontology fields (cell type is one that certainly could have researchers adding specifications, tissue is another). And EFO is a very responsive ontology with the ability to add terms in ~1 mo, so we should instead guide the development of that ontology instead of making exceptions.

**Comment by brianraymor** [comment]
That will require thought into how frequently we update the schema based on updates to the pinned ontologies too.

**Comment by brianraymor** [comment]
Removing "docs" label since the updated comment would change the validation requirements.


@jahilton -

I will also be adding your recommendations from https://github.com/chanzuckerberg/single-cell-curation/issues/235. 

Is your suggestion:

If there is not an exact match for the assay, clarifying text MAY be enclosed in parentheses and appended to the most accurate term. For example, the sci-plex assay could be curated as "EFO:0010183 (sci-plex)".

to simply:

If there is not an exact match for the assay, the most accurate term MUST be used.

---

Is there a common parent that could be specified for _most accurate child of_ such as [experimental process](http://www.ebi.ac.uk/efo/EFO_0002694) ?

**Comment by jahilton** [comment]
Yes, I think simplifying the guidance to use the most accurate term is good, along with the #235 addition.

I think experimental process is a safe ancestor to peg. In theory, [single cell library construction](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0010183) would work. My only concern is that an assay term gets added in a less-than-complete manner and either a submitter doesn't see it because they've narrowed their view or it doesn't validate because the term's not on the correct branch.

**Comment by brianraymor** [comment]
@jahilton - reviewing the 2.0.0 requirements in closer detail, I think we can simply remove:    

  _If there is not an exact match for the assay, clarifying text MAY be enclosed in parentheses and appended to the most accurate term. For example, the sci-plex assay could be curated as <code>"EFO:0010183 (sci-plex)"</code>.<br>_

because there's already guidance about parent terms or their most accurate child, resulting in:




### assay_ontology_term_id

<table><tbody>
    <tr>
      <th>Key</th>
      <td>assay_ontology_term_id</td>
    </tr>
    <tr>
      <th>Annotator</th>
      <td>Curator</td>
    </tr>
    <tr>
      <th>Value</th>
        <td>categorical with <code>str</code> categories. This MUST be an EFO term and either:<br><br>
          <ul><li>
            <a href="http://www.ebi.ac.uk/efo/EFO_0002772"><code>"EFO:0002772"</code></a> for <i>assay by molecule</i> or <b>preferably</b> its most accurate child
          </li>
          <li>
            <a href="http://www.ebi.ac.uk/efo/EFO_0010183"><code>"EFO:0010183"</code></a>  for <i>single cell library construction</i> or <b>preferably</b> its most accurate child
          </li></ul>
        An assay based on 10X Genomics products SHOULD either be <a href="http://www.ebi.ac.uk/efo/EFO_0008995"><code>"EFO:0008995"</code></a> for <i>10x technology</i> or <b>preferably</b> its most accurate child. An assay based on <i>SMART (Switching Mechanism at the 5' end of the RNA Template) or SMARTer technology</i> SHOULD either be <a href="http://www.ebi.ac.uk/efo/EFO_0010184"><code>"EFO:0010184"</code></a> for <i>Smart-like</i> or preferably its most accurate child.<br><br>
<br>Recommended values for specific assays:
          <br><br>
          <table>
          <thead>
          <tr>
          <th>For</th>
          <th>Use</th>
          </tr>
          </thead>
          <tbody>
            <tr>
              <td><i>10x 3' v2</i></td>
              <td><a href="http://www.ebi.ac.uk/efo/EFO_0009899"><code>"EFO:0009899"</code></a></td>
            </tr>
            <tr>
              <td><i>10x 3' v3</i></td>
              <td><a href="http://www.ebi.ac.uk/efo/EFO_0009922"><code>"EFO:0009922"</code></a></td>
            </tr>
            <tr>
              <td><i>10x 5' v1</i></td>
              <td><a href="http://www.ebi.ac.uk/efo/EFO_0011025"><code>"EFO:0011025"</code></a></td>
            </tr>
            <tr>
              <td><i>10x 5' v2</i></td>
              <td><a href="http://www.ebi.ac.uk/efo/EFO_0009900"><code>"EFO:0009900"</code></a></td>
            </tr>
            <tr>
              <td><i>Smart-seq</i></td>
              <td><a href="http://www.ebi.ac.uk/efo/EFO_0008930"><code>"EFO:0008930"</code></a></td>
            </tr>
            <tr>
              <td><i>Smart-seq2</i></td>
              <td><a href="http://www.ebi.ac.uk/efo/EFO_0008931"><code>"EFO:0008931"</code></a></td>
            </tr>
          </tbody></table>
        </td>
    </tr>
</tbody></table>
<br>

**Comment by jahilton** [comment]
Yes, for documentation updates, I agree with just removing that text

**Comment by brianraymor** [comment]
Addressed in #252
